packages/ariakit/src/checkbox/__examples__/checkbox/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { Checkbox } from "ariakit/checkbox";
+import "./style.css";
+
+export default function Example() {
+  return (
+    <label className="label">
+      <Checkbox className="checkbox" />
+      Checkbox
```

Comment:
Let's use a more real-world example. Maybe this?
```suggestion
      I agree to terms and conditions
```

Reply:
i updated this.